library/src/main/java/net/yslibrary/licenseadapter/internal/LibrariesHolder.java

```diff
@@ -46,7 +46,7 @@ public void run() {
             cachedLicenses.put(library, license);
             notify(license, null);
           } catch (Exception e) {
-            notify(null, e);
+            notify(library.getLicense(), e);
```

Comment:
We need to return license anyway, so that we can handle error state properly.
`Library#getLicense` is annotated with `@Nonnull`, so this shouldn't be a problem.

https://github.com/yshrsmz/LicenseAdapter/blob/83bf4713556bbb00171e3dcb5e0446e10c09634d/library/src/main/java/net/yslibrary/licenseadapter/internal/LicenseViewHolder.java#L56-L67

Reply:
Ohhhhhh, I see. Then can you update the `onComplete` to have it's `license` param annotated with `@NonNull`?

Reply:
yes, we need non-null License instance to display error :)